packages/lesswrong/lib/collections/users/schema.ts

```diff
@@ -76,6 +76,16 @@ const schema: SchemaType<DbUser> = {
   emails: {
     type: Array,
     optional: true,
+    onCreate: ({document: user}) => {
```

Comment:
I don't actually think this thing currently works. It's a pain to test, but I just dug out an old Google account and this doesn't seem to have set it properly, so I guess, maybe I should just remove it?

Reply:
Ah yes, this is because of Vulcan's mess with user-creation callbacks being a separate case that awkwardly half mirrors the callbacks in the mutator, but not being kept in sync.

Reply:
Neither the current user callback handling, nor the current OAuth handling, will be with us for much longer.

Reply:
Yeah, I will leave this in as a reminder then, but add a comment that it's dead code

Reply:
```suggestion
    // This is dead code and doesn't actually run, but we do have to implement something like this in a post Meteor world
    onCreate: ({document: user}) => {
    
```